src/adapters/postgres/user-adapter.ts

```diff
@@ -256,9 +256,33 @@ export class PostgresUserService {
       whereClause.username = username;
     }
     let userDetails = await this.usersRepository.findOne({
-      where: whereClause
+      where: whereClause,
+      select: ["userId", "username", "name", "role", "district","state","mobile"]
     })
-    return userDetails;
+
+    const tenentDetails = await this.allUsersTenent(userDetails.userId)
+      let userData = {
```

Comment:
@souravbhowmik1999

can not you have code like this?
userDetails['tenantData'] = tenentDetails;
    return userDetails;